socorro/external/es/supersearch.py

```diff
@@ -481,7 +482,15 @@ def get(self, **kwargs):
                     hits.append(self.format_fields(hit.to_dict()))
 
                 total = search.count()
-                aggregations = self.format_aggregations(results.aggregations)
+
+                aggregations = getattr(results, 'aggregations', {})
+                if aggregations:
```

Comment:
No need. If it *is* an empty dict, the `format_aggregations` should just be iterating of an empty dict.